projects/canopy/src/lib/carousel/carousel.component.ts

```diff
@@ -41,13 +44,32 @@ export class LgCarouselComponent implements AfterContentInit, OnDestroy {
   pause = new BehaviorSubject<boolean>(false);
   pausableTimer$: Observable<void>;
 
+  @HostListener('document:keydown.tab', ['$event'])
+  onKeydownHandler(event: KeyboardEvent) {
+    const target = event.target as HTMLElement;
+    const parent = target.parentElement as HTMLElement;
+    const el = parent.className;
+
+    if (el === 'lg-carousel-item') {
```

Comment:
I'm not sure about this. Doesn't this programmatically change the tab order of the document? Furthermore, if there are two elements that can receive focus on the `lg-carousel-item`, for example two buttons, doesn't this prevent the second element from ever being able to be focused on? Effectively making it only accessible to customers with a mouse?

Reply:
(thinking out loud) could an alternative be an extension of what you've done with the pausing when a button is clicked? I.e. whilst an element within the `lg-carousel-item` has focus, the carousel is paused (and restarts when focus is lost [if it was auto-paused])?